enterprise/security/src/main/java/org/neo4j/server/security/enterprise/auth/RoleManager.java

```diff
@@ -42,4 +44,10 @@
      * @throws IOException
      */
     void removeUserFromRole( String username, String roleName ) throws IOException;
+
+    List<String> getAllRoleNames();
```

Comment:
I think it'll still clarify the interface by making type say more about the content.